### PR TITLE
added browser navigation for java

### PR DESCRIPTION
**Pull request by rpallavisharma**
## **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

added browser navigation for java
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

## **Type**
enhancement, documentation

___

## **Description**
- Added a new Java test class `browserNavigation` demonstrating various browser navigation methods.
- Updated Java code examples for browser navigation in the Selenium documentation for English, Japanese, Brazilian Portuguese, and Simplified Chinese.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>NavigationTest.java</strong><dd><code>Add Browser Navigation Test for Java</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java

<li>Renamed class from <code>BaseTest</code> to <code>browserNavigation</code>.<br> <li> Added <code>navigateBrowser</code> test method demonstrating browser navigation <br>methods: <code>get</code>, <code>navigate().to</code>, <code>navigate().back</code>, <code>navigate().forward</code>, and <br><code>navigate().refresh</code>.<br> <li> Asserted page title to verify navigation success.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1658/files#diff-469552dff1badf9df3b13350a838e1d25b6afdee575e9800dcdc47b89a195a63">+35/-2</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>navigation.en.md</strong><dd><code>Update Java Browser Navigation Examples in English Documentation</code></dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/interactions/navigation.en.md

- Updated Java code examples for browser navigation methods.

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1658/files#diff-a656c7e9e1dc8bd63d5b0cb72ec60b2b8f165916103588c80e71302e95bd31ac">+16/-13</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>navigation.ja.md</strong><dd><code>Update Java Browser Navigation Examples in Japanese Documentation</code></dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/interactions/navigation.ja.md

<li>Updated Java code examples for browser navigation methods in Japanese <br>documentation.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1658/files#diff-d9517ab3b5e5dcec6055ed1833b6b968c673baf55ca106c52a0ae80a6aac2307">+15/-11</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>navigation.pt-br.md</strong><dd><code>Update Java Browser Navigation Examples in Brazilian Portuguese </code><br><code>Documentation</code></dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/interactions/navigation.pt-br.md

<li>Updated Java code examples for browser navigation methods in Brazilian <br>Portuguese documentation.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1658/files#diff-4ee33bad3b62d1d7f4c9a78747817b85e7d901f4aae0735b8ef640e4bec081fd">+13/-11</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>navigation.zh-cn.md</strong><dd><code>Update Java Browser Navigation Examples in Simplified Chinese </code><br><code>Documentation</code></dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/interactions/navigation.zh-cn.md

<li>Updated Java code examples for browser navigation methods in <br>Simplified Chinese documentation.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1658/files#diff-2e8c23b089e0f617b45b4a28305835ff439897baa3fada7df161ed3444b11c50">+12/-10</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java [hunk 1]
```diff
@@ -1,7 +1,40 @@
 package dev.selenium.interactions;
 
-import dev.selenium.BaseTest;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class NavigationTest extends BaseTest {
+public class browserNavigation {
```
Comment (harsha509) [inline examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java]: `public class NavigationTest {`

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI @pallavigitwork ,

class name should be same as file in here!

File: examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java (hunk 1 above)
Comment (rpallavisharma) [reply examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java]: Yes thats obvious. Not sure why this code is showing . I had changed . Will check and update in sometime . Again.

Comment (rpallavisharma) [reply examples/java/src/test/java/dev/selenium/interactions/NavigationTest.java]: corrected the mistake. please check. sorry about this.

**Review by harsha509 — APPROVED** [review]
Thank you @pallavigitwork !

**Pull request merged**